juriscraper/pacer/acms_docket.py

```diff
@@ -32,6 +34,24 @@ def _parse_text(self, text):
         """
         self._acms_json = json.loads(text)
 
+    def query(self, case_id: str):
+        """
+        Queries the court API for case and docket details for a given case ID.
+
+        :param case_id: The unique identifier of the case to query.
+        """
+        # Fetch Case Details
+        case_data = self.api_client.get_case_details(case_id)
```

Comment:
Aren’t there more query parameters available for ACMS, like `dateFrom` or `dateTo`, as in other appellate courts?

Is it only possible to retrieve the entire case?

Reply:
> Aren’t there more query parameters available for ACMS, like dateFrom or dateTo, as in other appellate courts?

I haven't found any exposed query parameters or user interface options (like dateFrom or dateTo fields) that allow for pre-filtering or customizing the 'full docket report' within the application before it loads.

In contrast to other appellate court systems, where users are typically presented with a form or filtering options to refine the report (e.g., by date ranges, checkbox to exclude parties) before the report is generated, ACMS currently redirects the user straight to the full docket report upon clicking the 'full docket report' button.

> Is it only possible to retrieve the entire case?

I think so.

Reply:
Got it! Thank you.